src/test/isolation2/sql/fts_segment_reset.sql

```diff
@@ -48,7 +59,7 @@ select dbid, role, preferred_role, status from gp_segment_configuration where co
 select gp_inject_fault('postmaster_delay_termination_bg_writer', 'reset', dbid) from gp_segment_configuration where role = 'p' and content = 0;
 
 -- The only table that should have been created successfully
-drop table fts_reset_t3;
+3:drop table fts_reset_t3;
```

Comment:
Why is this necessary?

Reply:
This is another point if instability. Running statements on a cluster where segments stopping, crashing, reset, etc may lead in broken client session, which is sometime happens for default session in this test. Default session is used in between of operations and can catch segment reset.
Session 3 is not used in the middle of the test and survives till the end, that's why this change introduced.